lib/engine.js

```diff
@@ -282,6 +282,7 @@ module.exports = function container (get, set, clear) {
               if (err) return cb(err)
               s.api_order = api_order
               order.status = api_order.status
+              if (api_order.reject_reason) order.reject_reason = api_order.reject_reason
```

Comment:
I think this is in line with [lib/engine.js#284](https://github.com/carlos8f/zenbot/blob/master/lib/engine.js#L284). It is more consistent and should be kept (see below).